### PR TITLE
[mssql-linux] Added mssql backup dir env var and Updated README and NOTES

### PR DESCRIPTION
**Pull request by thomasliddledba**
- Added feature to persist the backup directory variable in MSSQL

- Updated the README.md to reflect new feature for backup directory

- Updated README.md with markdown linting requirements (spaces...new line...etc)

- Updated NOTES.txt to remove hard-coded values with service port

### CODE REVIEW

File: stable/mssql-linux/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: SQL Server 2017 Linux Helm Chart
 name: mssql-linux
-version: 0.1.8
+version: 0.1.9
```
Comment (timstoop) [inline stable/mssql-linux/Chart.yaml]: You're adding a new feature (make use of an existing claim, nodeselector), so please bump the minor version of the Chart.

**Review by timstoop — CHANGES_REQUESTED** [review]
Aside from the chart version, looks fine.

Kind strange, if you told me 10 years ago I'd be willingly checking out how to deploy MSSQL, I'd laughed at your face. (Good change, though!)

File: stable/mssql-linux/Chart.yaml (hunk 1 above)
Comment (thomasliddledba) [reply stable/mssql-linux/Chart.yaml]: Good point.  Completed.

**Comment by thomasliddledba** [comment]
@timstoop - LOL.  As a DBA, if someone said to me to install MSSQL on linux I'd laugh.  Times are changing.  :)

**Pull request merged**